app/code/Magento/InventoryCatalog/view/adminhtml/ui_component/product_form.xml

```diff
@@ -66,6 +66,7 @@
                         <item name="source_code" xsi:type="string">source_code</item>
                         <item name="name" xsi:type="string">name</item>
                         <item name="qty" xsi:type="string">qty</item>
+                        <item name="source_status" xsi:type="string">enabled</item>
```

Comment:
Pls, check this string
According to `'source_status' => $row['source_status'],` data key should be `source_status`

Reply:
It's mapping from inventory source modal grid on product from. Field which represent status of source is named as "enabled" in this grid, look https://github.com/magento-engcom/msi/blob/develop/app/code/Magento/Inventory/view/adminhtml/ui_component/inventory_source_listing.xml#L137 .